### PR TITLE
Update error message - Closes #5017

### LINKED ISSUE #5017: Invalid error message in TxPool for per account limit
### Expected behavior

When `TxPool` reaches to per account transaction limit, it should show proper message. 

```
Transaction was not added because it exceeds maximum limit per account
```

### Actual behavior

Currently its showing error: 

```
Transaction was not added because of nonce or fee
```

### Steps to reproduce
- Sent more than 64 transactions from the same account. 

### Which version(s) does this affect? (Environment, OS, etc...)
4.0.0

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #5017 

### How was it solved?

Update the error message

### CODE REVIEW

File: elements/lisk-transaction-pool/src/transaction_pool.ts [hunk 1]
```diff
@@ -226,7 +226,7 @@ export class TransactionPool {
 				status: Status.FAIL,
 				errors: [
 					new TransactionPoolError(
-						'Transaction was not added because of nonce or fee',
+						'Transaction was not added because it exceeds maximum limit per account',
```
Comment (ManuGowda) [inline elements/lisk-transaction-pool/src/transaction_pool.ts]:
```suggestion
						`Transaction was not added to the pool as it exceeds maximum transaction limit per account`,
```

Comment (shuse2) [reply elements/lisk-transaction-pool/src/transaction_pool.ts]: I think `` is wrong?

Comment (ManuGowda) [reply elements/lisk-transaction-pool/src/transaction_pool.ts]: why?

Comment (shuse2) [reply elements/lisk-transaction-pool/src/transaction_pool.ts]: nvm i think i miss read

**Pull request merged**